descriptor.md

```diff
@@ -0,0 +1,123 @@
+<!--[metadata]>
++++
+draft = true
++++
+<![end-metadata]-->
```

Comment:
It was in the top of the other spec. Figured we were using it.

Was this copied over from docker? This looks like part of Docker's docs build system.